style.css

```diff
@@ -0,0 +1,794 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  width: auto;
+}
+
+ul {
+  list-style: none;
+}
+
+header {
+  height: 48px;
+  background: #3c3a39;
+  padding: 0 12px;
+  overflow: hidden;
+}
+
+#title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 80px;
+  line-height: 64px;
+  color: #ff6b00;
+  letter-spacing: 0.2px;
+  word-wrap: break-word;
+}
+
+#bottom-socials {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 25px 0;
+}
+
+@media only screen and (min-width: 768px) {
+  header {
+    padding: 0 100px;
+  }
+}
```

Comment:
- [ ] Please put all media query code in one media query block of code. This works without any issue but the trend is to put all media query code in a single block of code.